edb/lib/std/30-strfuncs.eql

```diff
@@ -19,11 +19,84 @@
 ## String functions
 
 
-# std::lower
-# ----------
+CREATE FUNCTION
+std::ascii(val: std::str) -> std::int64
```

Comment:
Given that this returns an integer Unicode codepoint, this should be called `ord()`.

Reply:
Done